app/medInria/main.cpp

```diff
@@ -79,7 +79,7 @@ int main(int argc,char* argv[]) {
     QLocale::setDefault(QLocale("C"));
 
     if (dtkApplicationArgumentsContain(&application, "-h") || dtkApplicationArgumentsContain(&application, "--help")) {
-        qDebug() << "Usage: MUSIC [--fullscreen|--no-fullscreen] [--stereo] "
+        qDebug() << "Usage: "<<QFileInfo(argv[0]).baseName().toStdString().c_str()<<" [--fullscreen|--no-fullscreen] [--stereo] "
```

Comment:
There is just this line left to replace with PROJECT_NAME

Reply:
In this particularly case, the "MUSIC" name is not always the name of the binary. This is the line to use to launch the software in a terminal

Reply:
Yes I didn't see that.